### PR TITLE
Return boolean through direct comparison

### PR DESCRIPTION
**Pull request by Goku-04**
**Changes**
* Remove ternary operator and simply rely on return value of strict equality (===) 

**Issues**
N/A

### CODE REVIEW

File: src/apps/experimental/components/library/GenresSectionContainer.tsx [hunk 1]
```diff
@@ -70,8 +70,8 @@ const GenresSectionContainer: FC<GenresSectionContainerProps> = ({
             centerText: true,
             cardLayout: false,
             shape: itemType === BaseItemKind.MusicAlbum ? 'overflowSquare' : 'overflowPortrait',
-            showParentTitle: itemType === BaseItemKind.MusicAlbum ? true : false,
-            showYear: itemType === BaseItemKind.MusicAlbum ? false : true
+            showParentTitle: (itemType === BaseItemKind.MusicAlbum),
+            showYear: (itemType !== BaseItemKind.MusicAlbum)
```
Comment (thornbill) [inline src/apps/experimental/components/library/GenresSectionContainer.tsx:74]: The parentheses really aren't necessary

```suggestion
            showParentTitle: itemType === BaseItemKind.MusicAlbum,
            showYear: itemType !== BaseItemKind.MusicAlbum
```

**Comment by thornbill** [comment]
We should add this eslint rule to the config to prevent any other issues of this type being added in the future: https://eslint.org/docs/latest/rules/no-unneeded-ternary

**Comment by thornbill** [comment]
@Goku-04 would you care to add that eslint rule as part of this PR? Thanks!

File: .eslintrc.js [hunk 2]
```diff
@@ -63,6 +63,8 @@ module.exports = {
         '@typescript-eslint/no-shadow': ['error'],
         'no-throw-literal': ['error'],
         'no-trailing-spaces': ['error'],
+        'no-unneeded-ternary': ['error'],
+        '@typescript-eslint/no-unneeded-ternary': ['error'],
```
Comment (thornbill) [inline .eslintrc.js:67]: This line isn't needed.

There are some rules that need to be specifically enabled for typescript and disabled in the default rules due to syntax differences in typescript vs javascript. This is not one of those so only the default rule needs enabled. 🙂

Comment (Goku-04) [reply .eslintrc.js:67]: so remove the 67th line?

Comment (thornbill) [reply .eslintrc.js:67]: yup

Comment (Goku-04) [reply .eslintrc.js]: done

**Comment by Goku-04** [comment]
The lint rule created some erros in other directories and I think some of them are false errors.

**Comment by dmitrylyzo** [comment]
> The lint rule created some erros in other directories and I think some of them are false errors.

All of them (9) are using unnecessary ternary: `x = condition ? true : false` => `x = condition`. In some cases `!!` (double negation) may be required to cast to boolean.

File: src/components/playback/playersettingsmenu.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ function showQualityMenu(player, btn) {
         const bitrate = parseInt(id, 10);
         if (bitrate !== selectedBitrate) {
             playbackManager.setMaxStreamingBitrate({
-                enableAutomaticBitrateDetection: bitrate ? false : true,
+                enableAutomaticBitrateDetection: !!bitrate,
```
Comment (dmitrylyzo) [inline src/components/playback/playersettingsmenu.js:49]:
```suggestion
                enableAutomaticBitrateDetection: !bitrate,
```

**Review by thornbill — APPROVED** [review]

**Pull request merged**